patches/api/0391-Fix-wrong-position-for-bed-spawn-loc.patch

```diff
@@ -0,0 +1,92 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Lulu13022002 <41980282+Lulu13022002@users.noreply.github.com>
+Date: Wed, 20 Jul 2022 14:09:22 +0200
+Subject: [PATCH] Fix wrong position for bed spawn loc
+
+
+diff --git a/src/main/java/org/bukkit/OfflinePlayer.java b/src/main/java/org/bukkit/OfflinePlayer.java
+index a7d1f1e701f23e851f735584a30bedadb0d8b9bd..1490ce660285bff5f8cd9c8bbc9ae7f2e642bde9 100644
+--- a/src/main/java/org/bukkit/OfflinePlayer.java
++++ b/src/main/java/org/bukkit/OfflinePlayer.java
+@@ -172,15 +172,29 @@ public interface OfflinePlayer extends ServerOperator, AnimalTamer, Configuratio
+      */
+     public boolean hasPlayedBefore();
+ 
++    // Paper start
+     /**
+-     * Gets the Location where the player will spawn at their bed, null if
+-     * they have not slept in one or their current bed spawn is invalid.
++     * Gets the Location where the player will spawn at their spawn block (bed, respawn anchor...), null if
```

Comment:
If we are editing javadocs anyway, this should mention that the method will not do a proper lookup by default.